data/tlg0008/tlg001/tlg0008.tlg001.perseus-eng3.xml

```diff
@@ -35,16 +34,17 @@
 						<monogr>
 							<author>Athenaeus</author>
 							<title>The Deipnosophists</title>
-							<respStmt>
-								<resp>with an English Translation by</resp>
-								<name>Charles Burton Gulick</name>
-							</respStmt>
+							<editor role="translator">
+								<persName>
+									<name>Charles Burton Gulick</name>
+								</persName>
+							</editor>
 							<imprint>
 								<pubPlace>Cambridge, MA</pubPlace>
 								<publisher>Harvard University Press</publisher>
 								<pubPlace>London</pubPlace>
 								<publisher>William Heinemann Ltd.</publisher>
-								<date>1927</date>
+								<date>1927-1941</date>
 							</imprint>
 							<biblScope type="volume">1-7</biblScope>
```

Comment:
<biblScope unit="volume">

Reply:
Hey @lcerrato so to resolve this do I need to change back the biblScope in these files to what it was previously? I'm afraid I'm not sure how exactly what I need to change  to make these files pass.  I did notice about the schema issue, but hoped that if I was only changing headers and other basic metadata that it wouldn't be an issue.   Sorry if I created an unnecessary issue.

Reply:
How did I not see that? Thanks and I will change that!